django-server/server/api/views.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import viewsets
+
+from .serializers import CafeteriaSerializer
+
+from .models import Cafeteria
+
+
+class CafeteriaViewSet(viewsets.ModelViewSet):
+    queryset = Cafeteria.objects.all().order_by('id')
+    serializer_class = CafeteriaSerializer
```

Comment:
Capacity is a private property and should not appear in get requests for all. Can we have different serializers for public and private properties? It would be best to show all properties to roles "cafeteria owner" and "admin".

Reply:
marked to be discussed in the near  future, when it will be obvious how to configure permissions

Reply:
marked as TODO in code in serializers of the apu

Reply:
Opened #55 so we don't miss that.